### ISSUE TITLE
Miner block height

### ISSUE THREAD

**Issue by feraldruid22** [nhatfield/hnt_monitor#88]
How do you get the block height and cpu core temperatures per miner.

The block height only available as a HNT metric coming from their system not the nodes. CPU is not available at all 
Mine is nebra I have added this to the hnt_monitor.yml

```

    environment:
      DO_NOT_REMOVE: "setup"
      HNT_HOTSPOT_MONITOR: "true"
      HNT_NEBRA_MONITOR: "true"

```

Also what is the point of having the internal LAN ips defined for Nebras?

**Comment by ubiru** [comment]
this also doesn't show on my hnt_monitor for nebra hotspot

**Comment by nhatfield** [comment]
There was a bug fix for nebra that was fixed in release 7.9.2... 7.10.0 is available now which contains arm support and reward earning fix

**Comment by nhatfield** [comment]
Also, the lan ip is needed because the collector hits the hotspot directly and pulls the diag json from it
